### ISSUE TITLE
html emits 0 response code for "default" responses

### ISSUE THREAD

**Issue by DavidBiesack** [swagger-api/swagger-codegen#4982]
##### Description

See [petstore output](https://github.com/swagger-api/swagger-codegen/blob/master/samples/html/index.html)
```
    <h3 class="field-label">Responses</h3>
    <h4 class="field-label">0</h4>
    successful operation
```
##### Swagger-codegen version

2.2.3-SNAPSHOT

##### Swagger declaration file content or url

petstore.yaml

bin/html-petstore.sh

##### Suggest a Fix

I'll submit a patch. isDefault in CodegenResponse is insufficient because it is true for 200 responses.
Needs a new member isDefaultCode which is only true if the response key is
```
  default: 
```

**Comment by wing328** [comment]
@DavidBiesack there's a similar fix (merged) for `html2`: https://github.com/swagger-api/swagger-codegen/pull/4914

**Comment by DavidBiesack** [comment]
Thanks, @wing328  - I'll abandon this PR and adopt the approach used on #4914 - it is simpler and has fewer changes

**Comment by DavidBiesack** [comment]
Fixed in #4991
